utils-core/main/default/lwc/datatable/datatable.js

```diff
@@ -289,6 +294,10 @@ export default class Datatable extends LightningElement {
         }
         this._isRendered = true;
         this._messageService = this.template.querySelector('c-message-service');
+        // Fixes display of row numbers beyond 99
+        if (this.checkboxType === 'Multi') {
+            this.template.querySelector('c-datatable-extension').setAttribute('show-row-number-column');
```

Comment:
nice

Reply:
Ah, i forgot to back that out - that actually doesn't work LOL. Next patch, I guess

I ended up having to add a property to the template.

Reply:
Haha well, now I’m _extra_ glad I commented! Good to know.